README.md

```diff
@@ -21,7 +21,11 @@ https://cloud.google.com/endpoints/docs/frameworks/java
 To install test versions to Maven for easier dependency management, simply run:
 
     gradle install
+    
+## Build pacakage
```

Comment:
Put this section above installing to local maven. Also, you have a typo in "package".